audit_exceptions/versioned_dependencies_conflicts_allowlist.json

```diff
@@ -3,6 +3,7 @@
   "hive",
   "libgaiagraphics",
   "librasterlite",
+  "opendht",
```

Comment:
I think this should be okay as `opendht` uses `gnutls` rather than `openssl@1.1`

Audit failure is due to dependency chain of `gnutls -> unbound -> openssl@1.1`